src/main/java/org/geysermc/discordbot/tags/TagsManager.java

```diff
@@ -124,7 +124,7 @@ private static void loadTags() {
                             }
 
                             if (content.toString().isEmpty()) {
-                                GeyserBot.LOGGER.warn("Tag '" + tagName + "' has empty content! Skipping tag.");
+                                GeyserBot.LOGGER.debug("Tag '" + tagName + "' has empty content!");
```

Comment:
You just changed the log message, I think you meant to remove `continue`

Reply:
oops ty